ivi/rohdeschwarz/rohdeschwarzBaseScope.py

```diff
@@ -1057,10 +1059,12 @@ def _set_trigger_edge_slope(self, value):
     def _get_trigger_source(self):
         if not self._driver_operation_simulate and not self._get_cache_valid():
             value = self._ask("trigger:a:source?").lower()
-            if value[0:2] == 'ch':
-                self._trigger_source = 'channel' + value[-1]
-            elif value[0] == 'd':
-                self._trigger_source == 'digital' + value[-1]
+            channel_number = str(re.findall(channel_re, value)[0])
```

Comment:
In line with the suggestion above, I suggest here:
```suggestion
            m = _channel_re.match(value)
```